dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/aot/NacosReflectionTypeDescriberRegistrar.java

```diff
@@ -0,0 +1,86 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.registry.nacos.aot;
+
+import org.apache.dubbo.aot.api.MemberCategory;
+import org.apache.dubbo.aot.api.ReflectionTypeDescriberRegistrar;
+import org.apache.dubbo.aot.api.TypeDescriber;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+
+public class NacosReflectionTypeDescriberRegistrar implements ReflectionTypeDescriberRegistrar {
+
+    @Override
+    public List<TypeDescriber> getTypeDescribers() {
+        List<TypeDescriber> typeDescribers = new ArrayList<>();
+        String[] classesWithDeclared = {"com.alibaba.nacos.api.ability.ClientAbilities", "com.alibaba.nacos.api.config.ability.ClientConfigAbility", "com.alibaba.nacos.api.config.remote.request.AbstractConfigRequest", "com.alibaba.nacos.api.config.remote.request.ConfigBatchListenRequest", "com.alibaba.nacos.api.config.remote.request.ConfigBatchListenRequest$ConfigListenContext", "com.alibaba.nacos.api.config.remote.request.ConfigPublishRequest", "com.alibaba.nacos.api.config.remote.request.ConfigQueryRequest", "com.alibaba.nacos.api.config.remote.response.ConfigChangeBatchListenResponse", "com.alibaba.nacos.api.config.remote.response.ConfigChangeBatchListenResponse$ConfigContext", "com.alibaba.nacos.api.config.remote.response.ConfigPublishResponse", "com.alibaba.nacos.api.config.remote.response.ConfigQueryResponse", "com.alibaba.nacos.api.naming.ability.ClientNamingAbility", "com.alibaba.nacos.api.naming.pojo.Instance", "com.alibaba.nacos.api.naming.pojo.ServiceInfo", "com.alibaba.nacos.api.naming.remote.request.AbstractNamingRequest", "com.alibaba.nacos.api.naming.remote.request.InstanceRequest", "com.alibaba.nacos.api.naming.remote.request.NotifySubscriberRequest", "com.alibaba.nacos.api.naming.remote.request.ServiceQueryRequest", "com.alibaba.nacos.api.naming.remote.request.SubscribeServiceRequest", "com.alibaba.nacos.api.naming.remote.response.InstanceResponse", "com.alibaba.nacos.api.naming.remote.response.NotifySubscriberResponse", "com.alibaba.nacos.api.naming.remote.response.QueryServiceResponse", "com.alibaba.nacos.api.naming.remote.response.SubscribeServiceResponse", "com.alibaba.nacos.api.remote.ability.ClientRemoteAbility", "com.alibaba.nacos.api.remote.request.ConnectionSetupRequest", "com.alibaba.nacos.api.remote.request.HealthCheckRequest", "com.alibaba.nacos.api.remote.request.InternalRequest", "com.alibaba.nacos.api.remote.request.Request", "com.alibaba.nacos.api.remote.request.ServerCheckRequest", "com.alibaba.nacos.api.remote.request.ServerRequest", "com.alibaba.nacos.api.remote.response.HealthCheckResponse", "com.alibaba.nacos.api.remote.response.Response", "com.alibaba.nacos.api.remote.response.ServerCheckResponse", "com.alibaba.nacos.common.remote.TlsConfig", "com.alibaba.nacos.common.remote.client.RpcClientTlsConfig"};
```

Comment:
We need to introduce class dependencies directly in order to prevent incompatibilities from going undetected once nacos is upgraded.

Reply:
Fixed in the latest commit.